extensions-core/histogram/src/main/java/io/druid/query/aggregation/histogram/ApproximateHistogramAggregator.java

```diff
@@ -59,7 +60,9 @@ public ApproximateHistogramAggregator(
   @Override
   public void aggregate()
   {
-    histogram.offer(selector.getFloat());
+    if (NullHandling.replaceWithDefault() || !selector.isNull()) {
```

Comment:
What does `NullHandling.replaceWithDefault()` mean here? Does that mean nulls have replaced with some default value at ingestion time? If so, I guess `select.isNull()` always return `false` and thus this check can be simplified into `if (!selector.isNull())`.

Reply:
this means whether nulls should be replaced with default value. 
In case of ExpressionColumnValueSelector isNull will compute the expression and then give the result, the check is there to not have any performance impact of calling isNull for default case.

Reply:
Got it. Thanks. Would you add a comment here?

Reply:
sure.